gradle/libs.versions.toml

```diff
@@ -46,6 +47,8 @@ gradlePlugin-shadow = "7.1.2"
 gradlePlugin-nexusPublish = "1.1.0"
 gradlePlugin-gradlePluginPublish = "0.20.0"
 gradlePlugin-gradleNode = "3.5.1"
+gradlePlugin-kotlinDsl= "2.4.1"
+gradlePlugin-pluginPublish = "0.20.0"
```

Comment:
Not relevant if the `gradle-plugin` convention is removed, but it's [not recommended](https://docs.gradle.org/current/userguide/kotlin_dsl.html#sec:kotlin-dsl_plugin) to specify the version of `kotlin-dsl`, and it looks like the dependency for `pluginPublish` is [already defined](https://github.com/Kotlin/dokka/pull/3074/files#diff-697f70cdd88ba88fe77eebda60c7e143f6ad1286bca75017421e93ad84fb87dfR118) below under the `[plugins]` section (unless these are two different dependencies with the same version)

Reply:
`kotlin-dsl` will be replaced with `expectedKotlinDslPluginsVersion`, see  https://github.com/gradle/gradle/issues/17016

> it looks like the dependency for pluginPublish is [already defined](https://github.com/Kotlin/dokka/pull/3074/files#diff-697f70cdd88ba88fe77eebda60c7e143f6ad1286bca75017421e93ad84fb87dfR118) below under the [plugins] section (unless these are two different dependencies with the same version)

Oh, I will remove it, for some reason the alias is not used in the runner project